pkg/gadgets/bindsnoop/types/types.go

```diff
@@ -0,0 +1,27 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright 2019-2022 The Inspektor Gadget authors
+
+package types
+
+import (
+	eventtypes "github.com/kinvolk/inspektor-gadget/pkg/types"
+)
+
+type Event struct {
+	eventtypes.Event
+
+	Pid       uint32 `json:"pid,omitempty"`
+	Comm      string `json:"comm,omitempty"`
+	Protocol  string `json:"proto,omitempty"`
+	Addr      string `json:"addr,omitempty"`
+	Port      uint16 `json:"port,omitempty"`
+	Options   string `json:"opts,omitempty"`
+	Interface int    `json:"if,omitempty"`
```

Comment:
I think there should be an `InterfaceName` field as the index is only meaningful locally and kubectl-gadget doesn't have a way to convert it to the name. (because it could be running on a different host).

Reply:
I did not understand your comment, can you please give me more details?